pkg/api/const.go

```diff
@@ -434,10 +435,23 @@ const (
 const (
 	//AzureADIdentitySystem Identity System
 	AzureADIdentitySystem = "azure_ad"
-	//ADFS Identity System
+	//ADFSIdentitySystem System
 	ADFSIdentitySystem = "adfs"
 )
 
+const (
+	//DependenciesLocationDefault dependencies are in AzurePublic cloud
+	DependenciesLocationDefault = "default"
+	//DependenciesLocationChina dependencies are in AzureChina cloud
+	DependenciesLocationChina = "china"
+	//DependenciesLocationGerman dependencies are in AzureGerman cloud
+	DependenciesLocationGerman = "german"
+	//DependenciesLocationUSGovernment dependencies are us_government cloud
+	DependenciesLocationUSGovernment = "us_government"
+	//DependenciesLocationNone dependencies are in the VHD
+	DependenciesLocationNone = "none"
```

Comment:
VHD is not its own cloud location. There is an overlap between cloud and pre-pulling dependencies on as VM image. It could be public cloud + VHD, China cloud + VHD, etc.

Also, I do not see `DependenciesLocationNone` being used anywhere. We shouldn't add a constant unless it is used in code.